src/main/java/org/folio/helper/OkapiModulesClientHelper.java

```diff
@@ -15,11 +15,9 @@
 
 public class OkapiModulesClientHelper {
 
-  private OkapiModulesClientHelper() {
-    throw new IllegalStateException("Utility class");
-  }
+  public OkapiModulesClientHelper() { }
 
-  public static NotifySendRequest buildNotifySendRequest(TemplateProcessingResult result, PatronNoticeEntity entity) {
```

Comment:
Why do methods in this class have to be instance methods?

Reply:
Static methods are bad for testability - they can't be mocked without tools like Powermock, the use of which is considered by some a sign of a bad code design. Powermock helps to test legacy code which can't be changed without massive refactoring, but in this case it was easier to change the implementation.

Reply:
Ok, thanks.